patroni/dcs/etcd.py

```diff
@@ -215,6 +215,13 @@ def get_etcd_client(config):
                 sleep(5)
         return client
 
+    def set_ttl(self, ttl):
+        ttl = int(ttl)
+        if self._ttl != ttl:
+            self.reset_cluster()
```

Comment:
I suppose setting the even should wake up the other members to re-read the new TTL from the DCS-based configuration, but why reset_cluster()?